src/main/java/com/google/codeu/data/TeamDatastore.java

```diff
@@ -0,0 +1,61 @@
+package com.google.codeu.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.UUID;
+import com.google.codeu.data.Team;
+
+public class TeamDatastore {
```

Comment:
It may be helpful to write helper methods to convert from an entity to a team and vice versa:

```java

Team getTeam(Entity entity);
Entity get Entity(Team team);
```
It'll help reduce repetition since you'll be doing that conversion often.